app/components/find/results/search_result_component.rb

```diff
@@ -24,6 +25,10 @@ def has_sites?
         @has_sites
       end
 
+      def location_label
+        @sites_count > 1 ? t('.nearest_location') : t('.location')
```

Comment:
Have we considered moving this to the translations layer using the [pluralizations](https://guides.rubyonrails.org/i18n.html#pluralization)?